tee-worker/litentry/core/parachain-extrinsic-task/receiver/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(all(feature = "std", feature = "sgx"))]
+compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
+
+use itp_extrinsics_factory::CreateExtrinsics;
+use itp_ocall_api::EnclaveOnChainOCallApi;
+use itp_types::parentchain::ParentchainId;
+use lc_parachain_extrinsic_task_sender::init_parachain_extrinsic_sender_storage;
+use log::*;
+use std::{format, string::String, sync::Arc, time, vec};
+
+pub fn run_parachain_extrinsic_task_receiver<ExtrinsicsFactory, OCallApi>(
+	api: Arc<OCallApi>,
+	extrinsic_factory: Arc<ExtrinsicsFactory>,
+) -> Result<(), String>
+where
+	ExtrinsicsFactory: CreateExtrinsics + Send + Sync + 'static,
+	OCallApi: EnclaveOnChainOCallApi,
+{
+	let task_receiver = init_parachain_extrinsic_sender_storage().map_err(|e| {
+		format!("Failed to initialize parachain extrinsic task sender storage: {:?}", e)
+	})?;
+	let mut calls = vec::Vec::new();
+
+	loop {
+		let start_time = time::Instant::now();
+		while start_time.elapsed() < time::Duration::from_secs(6) {
```

Comment:
Any specific reason for 6? Or is it just a random number that we feel comfortable to start with?

Reply:
I forgot to ask about this, for now I put what was mentioned in the issue but I wasn't sure what a sensible value for this would be